src/gwt/src/org/rstudio/studio/client/workbench/prefs/model/UserPrefsAccessorConstants_fr.properties

```
@@ -404,6 +412,10 @@ toolbarVisibleDescription= Indiquer ou non, si la barre d''outils doit être aff
 defaultProjectLocationTitle= Emplacement par défaut du nouveau projet
 defaultProjectLocationDescription= Le chemin de répertoire sous lequel placer les nouveaux projets par défaut.
 
+# The default directory to use in file dialogs when opening a project.
+defaultOpenProjectLocationTitle = Emplacement du projet ouvert par défaut
```

```suggestion
defaultOpenProjectLocationTitle = Emplacement par défaut du projet ouvert
```

Less sure about this one